docs/sphinx/user_guide/feature/plugins.rst

```diff
@@ -0,0 +1,130 @@
+.. ##
+.. ## Copyright (c) 2016-20, Lawrence Livermore National Security, LLC
+.. ## and other RAJA project contributors. See the RAJA/COPYRIGHT file
+.. ## for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+.. ##
+
+.. _plugins-label:
+
+========
+Plugins
+========
+
+------------
+About RAJA Plugins
+------------
+
+RAJA supports user-made plugins that may be loaded either at the time of compilation or during runtime. These two methods are not mutually exclusive, as plugins loaded statically can be run alongside plugins that are loaded dynamically.
+
+------------
+Using RAJA Plugins
+------------
+
+^^^^^^^^^^^^^^^^^
+Static vs Dynamic Linking
```

Comment:
The term 'linking' should be changed to 'loading' in the user docs and elsewhere.